reference/employer/openapi.yaml

```diff
@@ -0,0 +1,1083 @@
+                              nullable: true
+                            last_name:
+                              type: string
+                              description: The legal last name of the individual.
+                            emails:
+                              type: array
+                              nullable: true
+                              items:
+                                type: object
+                                properties:
+                                  data:
+                                    type: string
+                                  type:
+                                    type: string
+                                    enum:
+                                      - work
+                                      - personal
+                            phone_numbers:
+                              type: array
+                              nullable: true
+                              items:
+                                type: object
+                                properties:
+                                  data:
+                                    type: string
+                                    pattern: '[0-9]{10}'
+                                  type:
+                                    type: string
+                                    enum:
+                                      - work
+                                      - personal
+                            dob:
+                              $ref: '#/components/schemas/Date'
+                            residence:
+                              $ref: '#/components/schemas/Location'
+              examples:
+                Success:
+                  value:
+                    responses:
+                      - individual_id: 5d0b10a1-a09a-430f-81f1-20be735dc5e9
+                        code: 200
+                        body:
+                          id: 5d0b10a1-a09a-430f-81f1-20be735dc5e9
+                          first_name: Jane
+                          middle_name: null
+                          last_name: Doe
+                          emails:
+                            - data: jane@acme.com
+                              type: work
+                            - data: janed@personal.com
+                              type: personal
+                          phone_numbers:
+                            - data: '5051234567'
+                              type: personal
+                          dob: '1970-01-01'
+                          residence:
+                            line1: 123 Main St
+                            line2: Apt C
+                            city: Schenectady
+                            state: NY
+                            postal_code: '12345'
+                            country: US
+      summary: Individual
+      operationId: get-individual
+      tags:
+        - Employer
+      requestBody:
+        $ref: '#/components/requestBodies/Individual-Ids'
```

Comment:
Do you need to somehow set the content-type to application/json? If I go to Try It > Body, I get a warning message `Content is a valid JSON object, but the Content-Type header is not set.`. 

This comment holds for all batch endpoints!